### PR TITLE
SentinelOne V2- 3.2.23

### PR DESCRIPTION
**Pull request by munna-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by merit-maita — COMMENTED** [review]
thank you for the contribution, it looks good!
let's please set up ca short demo.

**Comment by merit-maita** [comment]
Demo is set on Wednesday 11:30 AM

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 1]
```diff
@@ -1146,28 +1154,29 @@ def get_threats_command(client: Client, args: dict) -> CommandResults:
 
 def get_hash_command(client: Client, args: dict) -> CommandResults:
     """
-    Get hash reputation.
+    Get hash verdict.
```
Comment (kgal-pan) [inline Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py:1157]: As suggested in the demo, this command be replaced to use `!file` reputation command. See https://xsoar.pan.dev/docs/integrations/generic-commands-reputation and 
https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-filehash-reputation for more details.

**Review by merit-maita — APPROVED** [review]

**Comment by DavinGeorgeSentinelOne** [comment]
Hi Team, do we know when this will be released please?

**Comment by kgal-pan** [comment]
@DavinGeorgeSentinelOne - It's already out, see https://cortex.marketplace.pan.dev/marketplace/details/SentinelOne:

image

**Pull request merged**